tracer/test/test-applications/integrations/Samples.OpenTelemetrySdk/Program.cs

```diff
@@ -67,13 +87,23 @@ public static async Task Main(string[] args)
             using (otherSpan = _otherLibraryTracer.StartActiveSpan("Response"))
             {
                 PrintSpanStartedInformation(otherSpan);
+                
+#if OTEL_1_5
+                logger.LogInformation("Response from other library");
```

Comment:
If you're trying to test "from another library", you should get a logger with a different name. And then we can see that the OTLP logs scope changes